runtime/convertValues_test.go

```diff
@@ -3912,48 +3839,6 @@ func TestRuntimePublicKeyImport(t *testing.T) {
 						cadence.NewUInt8(0),
 					},
 				).WithType(SignAlgoType),
-
-				cadence.NewBool(true),
-			},
-		).WithType(PublicKeyType)
-
-		storage := newTestLedger(nil, nil)
-
-		runtimeInterface := &testRuntimeInterface{
-			storage: storage,
-			decodeArgument: func(b []byte, t cadence.Type) (value cadence.Value, err error) {
-				return json.Decode(b)
-			},
-		}
-
-		_, err := executeScript(t, script, publicKey, runtimeInterface)
-		require.Error(t, err)
-
-		var argErr *InvalidEntryPointArgumentError
-		require.ErrorAs(t, err, &argErr)
-	})
-
-	t.Run("Invalid content in public key", func(t *testing.T) {
```

Comment:
I _think_ deleting this makes sense because it's no longer possible for `PublicKey` arguments to be invalid.

The test was failing because it was getting the new panic message I created `"invalid public key"`. For some reason this happened even when I used the `validatePublicKey` test code that I use elsewhere.

Reply:
Same here: Don't remove the test and keep it. The test asserts that the checking of the arguments that are sent along with scripts/transactions are properly validated. The invalid input here (strings in the key) should still result in `InvalidEntryPointArgumentError`s